cg/meta/upload/nipt/nipt.py

```diff
@@ -24,10 +24,11 @@ def __init__(self, config: CGConfig):
         self.sftp_user: str = config.fluffy.sftp.user
         self.sftp_password: str = config.fluffy.sftp.password
         self.sftp_host: str = config.fluffy.sftp.host
+        self.port = config.fluffy.sftp.port
```

Comment:
I guess it would be consistent to name the member variables as the others?
```suggestion
        self.sftp_port = config.fluffy.sftp.port
```